### PR TITLE
Dataset

### PR DESCRIPTION
**Pull request by max-ksoll**

### CODE REVIEW

File: src/PyModel/fuxi_ligthning.py [hunk 1]
```diff
@@ -60,52 +45,34 @@ def __init__(
...
-        logger.debug("Setting Val Dataloader")
-        return DataLoader(
-            FuXiDataset(
-                self.val_ds_path, self.val_mean_ds_path, self.autoregression_steps
-            ),
-            **get_dataloader_params(self.batch_size),
-        )
-
-    def forward(self, *args: Any, **kwargs: Any) -> Any:
+    def forward(self, *args: Any, **kwargs: Any) -> torch.Tensor:
```
Comment (philippgri11) [inline src/PyModel/fuxi_ligthning.py:56]: Ich würde mir hier die Logik nochmal anschauen. Fine es ein wenig kommisch, dass wir immer den loss mitberechnen und ihn dann weg schmeißen.

**Review by philippgri11 — APPROVED** [review]

File: lightning_training.py [hunk 2]
```diff
@@ -80,7 +73,17 @@ def train():
             max_epochs=config.get('max_epochs', None)
         )
 
-        trainer.fit(model=model)
+        dm = FuXiDataModule(
+            data_dir="/Users/ksoll/git/FuXiClimatePrediction/data",
```
Comment (philippgri11) [inline lightning_training.py]: aus env

**Review by philippgri11 — APPROVED** [review]
